presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixConfigurationInitializer.java

```diff
@@ -60,7 +61,9 @@
     // Configs below are dependent on node joining the cluster
     private volatile boolean cacheReady;
     private boolean isMaster;
+    @Nullable
```

Comment:
Technically, `null` value is not legal for these once `updateConfiguration` is called. I think this can be further refactored so that Rubix is initialized when `RubixConfigurationInitializer` is constructed. This would remove all `cacheReady` ambiguity.